### PR TITLE
Task-01

### PR DESCRIPTION
**Pull request by n1cas**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+const repl = require("repl");
+
+function getRandomNumber (max) {
+   return Math.floor(1 + Math.random() * (+max + 1 - 1));
```
Comment (Draakan) [inline src/index.js]: what is the point of adding 1 and then subtracting it?

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+const repl = require("repl");
+
+function getRandomNumber (max) {
```
Comment (Draakan) [inline src/index.js]: optionally you can add a check to ensure that a number has actually been provided

File: src/index.js (hunk 1 above)
Comment (n1cas) [reply src/index.js]: it's a mistake, already fixed.

File: src/index.js (hunk 2 above)
Comment (n1cas) [reply src/index.js]: done

**Review by Draakan — APPROVED** [review]

**Pull request merged**